app/views/schools/discord_server_roles.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= render "discord_header", locals: { tab: :discord_server_roles } do %>
+  <% if params[:cause].present? && params[:cause] == "DeletedRoles" && @sync_service.sync_ready? %>
+    <%= render "discord_roles_diff", locals: { sync_service: @sync_service } %>
```

Comment:
This top-level `if-else` in a view is a clear indicator that one part of this (probably the partial) is supposed to be an entirely different page.